src/cascadia/TerminalApp/TabRowControl.xaml

```diff
@@ -11,96 +11,117 @@
-                            </ResourceDictionary>
-                            <ResourceDictionary x:Key="Dark">
-                                <StaticResource x:Key="SplitButtonBackground"
-                                                ResourceKey="TabViewButtonBackground" />
-                                <StaticResource x:Key="SplitButtonForeground"
-                                                ResourceKey="TabViewButtonForeground" />
-                                <StaticResource x:Key="SplitButtonBackgroundPressed"
-                                                ResourceKey="TabViewItemHeaderBackgroundPressed" />
-                                <StaticResource x:Key="SplitButtonForegroundPressed"
-                                                ResourceKey="TabViewItemHeaderForegroundPressed" />
-                                <StaticResource x:Key="SplitButtonBackgroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderBackgroundPointerOver" />
-                                <StaticResource x:Key="SplitButtonForegroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderForegroundPointerOver" />
-                            </ResourceDictionary>
-                            <ResourceDictionary x:Key="HighContrast">
-                                <StaticResource x:Key="SplitButtonBackground"
-                                                ResourceKey="TabViewButtonBackground" />
-                                <StaticResource x:Key="SplitButtonForeground"
-                                                ResourceKey="TabViewButtonForeground" />
-                                <StaticResource x:Key="SplitButtonBackgroundPressed"
-                                                ResourceKey="TabViewItemHeaderBackgroundPressed" />
-                                <StaticResource x:Key="SplitButtonForegroundPressed"
-                                                ResourceKey="TabViewItemHeaderForegroundPressed" />
-                                <StaticResource x:Key="SplitButtonBackgroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderBackgroundPointerOver" />
-                                <StaticResource x:Key="SplitButtonForegroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderForegroundPointerOver" />
-                            </ResourceDictionary>
-                        </ResourceDictionary.ThemeDictionaries>
-                    </ResourceDictionary>
-                </mux:SplitButton.Resources>
-            </mux:SplitButton>
-        </mux:TabView.TabStripFooter>
+        <ContentPresenter x:Name="ElevatedShield"
+                          Grid.Column="0">
 
-    </mux:TabView>
+            <!--  EA18 is the "Shield" glyph  -->
+            <FontIcon x:Uid="ElevationShield"

```

Comment:
i've looked at three files and seen "UacShield" "ElevatedShield" and "ElevationShield". Should we converge on just one?